amqp/src/main/java/io/github/tcdl/msb/adapters/amqp/AmqpAdapterFactory.java

```diff
@@ -96,6 +94,19 @@ protected ConnectionFactory createConnectionFactory(AmqpBrokerConfig adapterConf
         if(virtualHost.isPresent()) {
             connectionFactory.setVirtualHost(virtualHost.get());
         }
+
+        try {
+            if (adapterConfig.useSSL()) {
+                LOG.info("Configuring adapter to use SSL...");
+                connectionFactory.useSslProtocol();
```

Comment:
this use DEFAULT_SSL_PROTOCOL. 
From this http://grepcode.com/file/repo1.maven.org/maven2/com.rabbitmq/amqp-client/3.5.3/com/rabbitmq/client/ConnectionFactory.java#ConnectionFactory.0DEFAULT_SSL_PROTOCOL

i see that it's hardcoded to  {code} private static final String DEFAULT_SSL_PROTOCOL = "TLSv1"; {code}

Are we ok with that? Newer protocols are TLS 1.1 and TLS 1.2.